contracts/leaser/src/cmd/quote.rs

```diff
@@ -218,7 +218,7 @@ where
             return Err(ContractError::ZeroDownpayment {});
         }
 
-        let borrow = self.liability.init_borrow_amount(downpayment_lpn);
+        let borrow = self.liability.init_borrow_amount(downpayment_lpn, None);
```

Comment:
this is not correct. 

we have to take `max_ltv` as an input quote parameter, otherwise the calculation would not be accurate

Reply:
we have agreed to fix it